scripts/apps/authoring/authoring/controllers/ChangeImageController.js

```diff
@@ -203,6 +203,19 @@ export function ChangeImageController($scope, gettext, notify, modal, _, api, $r
         return false;
     }
 
+    function extractEditableMetadata(metadata) {
+        return _.pick(metadata, [
+            'headline',
```

Comment:
Would be great to implement this without hardcoding the field names.

Reply:
any suggestion?

Reply:
It depends on how the saving workflow works. It could work as follows:
1. tab 1, editing something
2. clicking another tab triggers the save > new tabs is opened with latest values

If we don't want to save on every tab change, then I guess hardcoding the fields is alright since they are also probably hardcoded in the view as well. In such case when the same fields are hardcoded in multiple places, it would be helpful to leave a comment pointing to all places so it's easy to add another field and know all other places where it should be added as well.

Reply:
I created a constant with all those values.